cockatrice/src/window_main.cpp

```diff
@@ -288,6 +293,87 @@ void MainWindow::actAbout()
     ));
 }
 
+void MainWindow::actUpdate()
+{
+    //Request the newest builds
+    QUrl versionUrl(LATEST_FILES_URL);
+    response = netMan.get(QNetworkRequest(versionUrl));
+    connect(response, SIGNAL(finished()),
+            this, SLOT(fileListFinished()));
+}
+
+void MainWindow::fileListFinished() {
+    //Parse the version results into a JSON array
```

Comment:
Instead of a comment, can we put this into a method of something like `QJsonArray parseBintrayVersionInfo` ?

Also, MainWindow isn't the right class for this - it can be a static method on UpdateChecker